wiki/FAQ/en.md

```diff
@@ -56,161 +78,258 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
 
 ![Gameplay footage of Osu! Tatakae! Ouendan in Nintendo DS][ouendan image]
 
-You can see the circles on the bottom screen, and on the top screen you have the story. Each stage is basically a self-contained story about a person in trouble. That's where the _Ouendan_ (Cheer squad) comes in. Through the magical power of male cheer-leading, you have to help the people in trouble.
+_Gameplay footage of Osu! Tatakae! Ouendan in Nintendo DS_
```

Comment:
Footage makes it sound like that the image is a movie... I would remove "footage"